PYME/Analysis/points/objectMeasure.py

```diff
@@ -161,49 +161,3 @@ def calcEdgeDists(objects, objMeasures):
         minEdgeDists.append(ed)
 
     return np.array(minEdgeDists)
-
-def get_labels_from_image(inp, img):
```

Comment:
Why is this being deleted?

Reply:
I believe it duplicates `cluster_morphology.get_labels_from_image()`.

Reply:
Correct.

Reply:
I'd argue the other way round - the `cluster_morphology` version  duplicates this one. It may be fairly academic in any case. I'd like to refactor whatever the final version is out into somewhere more sensible, but kind of feel that this is the canonical version, rather than the other way around. Before refactoring, I'd like to know the use case for the extra `minimum_localizations` argument is in the `cluster_morphology` version. To me it seems you'd almost always be better off filtering the resulting tabular output on cluster size rather than building it in as an argument to `get_labels_from_image()`.